os/arch/arm/src/artik05x/src/artik05x.h

```diff
@@ -61,6 +61,11 @@
 #define ARTIK05X_PROCFS_MOUNTPOINT "/proc"
 #endif
 
+#ifdef CONFIG_SMARTFS_MULTI_ROOT_DIRS
+#undef CONFIG_ARTIK05X_AUTOMOUNT_USERFS_DEVNAME
+#define CONFIG_ARTIK05X_AUTOMOUNT_USERFS_DEVNAME "/dev/smart0p8d1"
```

Comment:
1. User sets userfs devname as ```/dev/smart0p8```. But CONFIG_SMARTFS_MULTI_ROOT_DIRS changes it as ```....d1``` without notice. Is it OK?
2. I don't know why we need to change a devname with this config.

Reply:
In smart_initialize devname is getting  changed , it appends "d1"  to the devname when CONFIG_SMARTFS_MULTI_ROOTS_DIRS is enabled.

Reply:
I checked. You are right.